### ISSUE TITLE
Paint across UDIMs

### ISSUE THREAD

**Issue by Klonkel** [armory3d/armorpaint#1445]
It'd be really nice to be able to paint across multiple UDIMs instead of painting on a single one at the time.

**Comment by Krysidian** [comment]
Yeah that feels incredibly essential. Otherwise working with UDIMs feels basically useless unless you only use generated materials. Painting between two UDIM seams will happen constantly when working and should be possible.

**Comment by luboslenco** [comment]
Basic support added at https://github.com/armory3d/armorpaint/commit/99156dfc86c7360a8c0149e404c00005ff6bda4e.

When using shared layers, the layer resolution should be manually set higher for now, as the shared layer will pack the udim tiles into itself.
